docs/pages/zkvm/sdk-benchmarking.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Nexus zkVM SDK Benchmark"
+lang: en-US
+description: "Benchmark your Nexus project using the SDK"
+image: "/nexus-head.png"
+---
+
+import Image from "next/image";
+
+import { Steps } from "nextra/components";
+
+## Reproducible Proving and Verification
+
+<Steps>
+
+This example demonstrates how to use the Nexus SDK to generate proving and verification keys, compile a guest program, prove its execution, and verify the proof. The example uses the `guest` package as an example guest program.
```

Comment:
I clicked "SDK Benchmarking" and came here. Now I'm confused because I didn't see anything about benchmarking in the first paragraph.

Reply:
Oh yeah, I should update this line.

Reply:
updated in [36f79a4](https://github.com/nexus-xyz/nexus-zkvm/pull/271/commits/36f79a4ffc509fe44f4c288746c674652d698a4b)